lib/komoju/client.rb

```diff
@@ -163,10 +170,8 @@ def initialize(client)
     end
 
     # List events
-    #
-    # @param collection_options: additional collection options to pass with the request
-    def list(collection_options = {})
```

Comment:
I think we should not remove this option 😥 Looks we need to hack heroics ... 😓

Ref: previous option here https://github.com/komoju/komoju-ruby/commit/f67e4a4ff4a8009f652c203c8619afb7a9bf80af#diff-971db54e7f0e2821723c84cb193cdde7L168